### ISSUE TITLE
Conflicting node versions

### ISSUE THREAD

**Issue by Strernd** [GuildOfWeavers/genSTARK#15]
```
yarn add v1.13.0
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error @guildofweavers/galois@0.2.0: The engine "node" is incompatible with this module. Expected version "12.3.x". Got "12.6.0"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
λ  stark-test nvm install 12.3
Downloading and installing node v12.3.1...
Downloading https://nodejs.org/dist/v12.3.1/node-v12.3.1-darwin-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with shasum -a 256
Checksums matched!
Now using node v12.3.1 (npm v6.9.0)
λ  stark-test yarn add @guildofweavers/genstark
yarn add v1.13.0
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error @guildofweavers/air-script@0.2.3: The engine "node" is incompatible with this module. Expected version "12.6.x". Got "12.3.1"
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by bobbinth** [comment]
Thanks for pointing this out! I'm working on a big update which I'm planning to release in a couple of days, and as a part of this update I'll synchronize node versions for all dependencies at v12.7.

**Comment by bobbinth** [comment]
I've just published v0.5.2 of the library where Node version for all dependencies are set at `>=12.7.x`. So, this should be fixed, but let me know if you are still having issues.
